tests/execution_engine/test_sqlalchemy_batch_data.py

```diff
@@ -35,7 +35,26 @@ def test_instantiation_with_table_name(sqlite_view_engine):
     assert type(batch_data.record_set_name) == str
     assert batch_data.record_set_name == "great_expectations_sub_selection"
 
-    assert batch_data.use_quoted_name == False
+    assert batch_data.use_quoted_name is False
+
+
+def test_instantiation_with_query(sqlite_view_engine, test_df):
+    test_df.to_sql("test_table_0", con=sqlite_view_engine)
+
+    query: str = "SELECT * FROM test_table_0"
+    # If create_temp_table=False, a new temp table should NOT be created
+    # noinspection PyUnusedLocal
+    batch_data: SqlAlchemyBatchData = SqlAlchemyBatchData(
+        execution_engine=sqlite_view_engine,
+        query=query,
+        create_temp_table=False,
+    )
+    assert len(get_sqlite_temp_table_names(sqlite_view_engine)) == 1
+
+
+def test_instantiation_with_selectable():
+    # Note Abe 20111119: Fill this in
+    pass
```

Comment:
Where did this come from? Weird that I don't see it in `develop`.

Reply:
It is not there any more, and I will delete it.  Must have been there before.